Contentstack.Core/Configuration/Config.cs

```diff
@@ -97,6 +97,7 @@ internal string getBaseUrl (LivePreviewConfig livePreviewConfig, string contentT
         internal string regionCode()
         {
             if (Region == ContentstackRegion.US) return "";
+            if (Region == ContentstackRegion.Azure_NA) return "azure-";
```

Comment:
@AnkurContentstack Please removed this line as following code returns the expected value.

Reply:
Firstly Enum is not allowed any special char like "-" so we can not add in below Enum
<img width="352" alt="Screenshot 2023-07-14 at 4 05 52 PM" src="https://github.com/contentstack/contentstack-dotnet/assets/139141060/a5bae69a-3c37-40ff-af12-d41cc19627c6">

Like we have full url is azure-eu-cdn.contentstack.com so we need to define in enum like **azure-eu** and this is not allow in enum because if **-**

so here what we have done azure is declared in enum and concatenate with below condition URL
<img width="425" alt="Screenshot 2023-07-14 at 4 10 17 PM" src="https://github.com/contentstack/contentstack-dotnet/assets/139141060/bde50a87-6e01-427c-a895-554674eb67dc">

so again one issue came because of Azure_EU and Azure_NA both URL starting with azure and after - we have set value in above function. Issue is like we can not declare two time same value for Azure 

so that's why here I have set static condition for NA

Logic for Get value for Region and concatenation with post URL:
<img width="944" alt="Screenshot 2023-07-14 at 4 12 26 PM" src="https://github.com/contentstack/contentstack-dotnet/assets/139141060/a82173f7-d0ab-4a84-bbdd-83a9d0046db5">